.github/actions/build-test-image/action.yml

```diff
@@ -55,7 +55,7 @@ runs:
         file: ./integration-tests/test.Dockerfile
         build-args: |
           BASE_IMAGE=${{ inputs.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ inputs.QA_AWS_REGION }}.amazonaws.com/test-base-image
-          IMAGE_VERSION=${{ steps.version.output.version }}
+          IMAGE_VERSION=${{ steps.version.outputs.version }}
```

Comment:
What were the effects of this problem? Have we been testing incorrect versions this whole time?